python/jiminy_py/src/jiminy_py/viewer.py

```diff
@@ -32,7 +33,8 @@ class Viewer:
 
     def __init__(self, robot, use_theoretical_model=False,
                  urdf_rgba=None, robot_index=0,
-                 backend=None, window_name='python-pinocchio', scene_name='world'):
+                 backend=None, window_name='python-pinocchio', scene_name='world',
+                 urdf_root_path = None):
```

Comment:
It think 'mesh_root_path' would be more appropriate.

Reply:
Besides,  'mesh_root_path' should come right after 'use_theoretical_model' IMHO.